common/src/chain/config/regtest.rs

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) 2022 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::{
+    convert::Infallible,
+    fmt::{Display, Formatter},
+    str::FromStr,
+};
+
+use crate::{
+    chain::{
+        block::timestamp::BlockTimestamp,
+        config::{decode_hex, StakePoolData, DEFAULT_INITIAL_MINT, MIN_STAKE_POOL_PLEDGE},
+        tokens::OutputValue,
+        transaction::Destination,
+        Genesis, PoolId, TxOutput,
+    },
+    primitives::{per_thousand::PerThousand, Amount},
+};
+use crypto::{
+    key::{PrivateKey, PublicKey},
+    vrf::{VRFPrivateKey, VRFPublicKey},
+};
+
+#[derive(Clone, Debug)]
+pub struct GenesisStakingSettings {
+    pool_id: PoolId,
+    stake_private_key: PrivateKey,
+    vrf_private_key: VRFPrivateKey,
+}
+
+impl GenesisStakingSettings {
+    pub fn new(settings: &str) -> Self {
+        let settings_parts: Vec<&str> = settings.split(',').map(|s| s.trim()).collect();
+
+        let pool_id = settings_parts
+            .iter()
+            .find_map(|s| s.strip_prefix("genesis_pool_id:"))
+            .or(Some(
+                "123c4c600097c513e088b9be62069f0c74c7671c523c8e3469a1c3f14b7ea2c4",
```

Comment:
If these are default values, please move them to constants.

Reply:
Cool, done.